### PR TITLE
update hapi and fix test

### PR DESCRIPTION
**Pull request by ScottDowne**

### CODE REVIEW

File: tests/selenium/single-page-thunderbird-paypal-test.js [hunk 1]
```diff
@@ -17,13 +17,13 @@ module.exports = function(driver, By, done) {
   driver.wait(function() {
     return driver.getCurrentUrl().then(function(url) {
       var result = true;
-      if (url.indexOf('http://localhost:3000/en-US/thunderbird/thank-you/') === -1) {
+      if (url.indexOf('http://localhost:3000/en-CA/thunderbird/thank-you/') === -1) {
```
Comment (ScottDowne) [inline tests/selenium/single-page-thunderbird-paypal-test.js]: My best guess is this never really passed. We would just close the test manually and run the next manually.

**Comment by ScottDowne** [comment]
The checklist for the hapi update:
11.0.0 to 12.0.0 https://github.com/hapijs/hapi/issues/2985

- remove all usage of `query.qs` which I cannot find any usage from us. So nothing to do.

13.0.0 to 13.0.0 https://github.com/hapijs/hapi/issues/3040

- password length limited to 32 characters or more. I cannot find any usage of this either, so nothing to do.

**Comment by ScottDowne** [comment]
@alicoding r?

**Comment by alicoding** [comment]
Don't we have to update other dependencies at all? like `Boom` and other ones?

**Comment by ScottDowne** [comment]
No mention of it in the above upgrade guides, and everything worked.

Things I suggest you could do to review this is:

- run the code and try to hit something I missed and the tests missed, and ensure it also works.
- look through the code and double check that we're not using the qs or password things.

**Comment by alicoding** [comment]
r+

**Pull request merged**